src/explore-education-statistics-admin/src/pages/edit-release/components/EditReleasePageTemplate.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ReactNode } from 'react';
+import DfeNavLink from '@admin/components/DfeNavLink';
+import PreviousNextLinks, {
+  PreviousNextLink,
+} from '@admin/components/PreviousNextLinks';
+import Page from '../../../components/Page';
+
+interface Props {
+  releaseId: string;
+  children: ReactNode;
+  publicationTitle: string;
+  previousLink?: PreviousNextLink;
+  nextLink?: PreviousNextLink;
+}
+
+const EditReleasePageTemplate = ({
+  releaseId,
+  publicationTitle,
+  children,
+  previousLink,
+  nextLink,
+}: Props) => {
+  return (
```

Comment:
To cut down on all of the duplication, we could request for the release here in an effect, then not render the children until the release has been retrieved:

```javascript
useEffect(() => {
  const release = await getTheRelease(releaseId);
  setRelease(release);
}, [releaseId]);
```

Then in the render:

```javascript
{release && children}
```

Reply:
I'm actually planning on having different data per section - at the moment it's just duplicated as grabbing a "Release" until I tackle the individual page and see its data requirements.  I suppose that we could still do some de-duping here if we wanted to and just use a generic type to fetch and then pass to the children as, I guess, render props, but probably more complex than it needs to be.